portfolio/src/main/webapp/script.js

```diff
@@ -51,17 +45,60 @@ function createCommentElement(commentJson){
     return commentContainer;
 }
 
-function getComments(){
-    fetch("/data")
+function removeAllChild(parent){
+    while (parent.firstChild) {
+        parent.removeChild(parent.lastChild);
+    }
+}
+
+function populateCommentsList(comments){
+    const commentList = document.getElementById('comments');
+    removeAllChild(commentList);
+    let entry = null;
+    for(let i in comments){
+        entry = document.createElement('li');
+        comment = createCommentElement(comments[i]);
+        entry.appendChild(comment);
+        commentList.insertBefore(entry, commentList.firstChild);
+    }
+}
+
+function getNewestComments(){
+    fetch("/data?comments=newest")
+    .then(response => response.json())
+    .then((comments) => {
+        populateCommentsList(comments);
+    });
+}
+
+function getCurrentComments(){
+    fetch("/data?comments=current")
     .then(response => response.json())
     .then((comments) => {
-        const commentList = document.getElementById('comments');
-        let entry = null;
-        for(let i in comments){
-            entry = document.createElement('li');
-            comment = createCommentElement(comments[i]);
-            entry.appendChild(comment);
-            commentList.appendChild(entry);
+        populateCommentsList(comments);
+    });
+}
+
+function nextComments(){
+    fetch("/data?comments=next&oldest="+oldestSent+"&newest="+newestSent)
```

Comment:
Instead of concatenating strings, prefer expression interpolation for better readability, see [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).